bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -52,7 +61,7 @@ logging.info("Clustering coinc triggers (exclusive of zerolag)")
 e = e.cluster(args.cluster_window)
 
 logging.info("Dumping foreground triggers")
-f = h5py.File(args.output_file, "w")
+f = h5py.fw(args.output_file)
```

Comment:
Yeah, this is what I mean. See the class defined above that does the compression automatically, while still allowing the keyword style assignment.